python/tvm/target/target.py

```diff
@@ -623,27 +620,13 @@ def create_llvm_options(cpu_ver, config):  # pylint: disable=unused-argument
         args = [s.replace("=", "@") for s in llvm_options.split()]
         return "--llvm-options=" + ",".join(args)
 
-    # TVM target attributes string
-    def create_tvm_options(cpu_ver, config):  # pylint: disable=unused-argument
```

Comment:
This essentially broke the `link-params` on Hexagon.  You added code to transfer target attributes to the executor, but here you removed the target attributes from the `hexagon` target.